ontology/yaml/resources/HVAC/entity_types/AHU.yaml

```diff
@@ -17,6 +17,14 @@
 ########################
 
 ### Single Zone Units ###
+AHU_SS_SDM:
+  guid: "d289ff35-4b60-42ba-b32e-00a679add978"
+  description: "AHU with supply air damper monitoring."
+  is_canonical: true
+  implements:
+  - AHU
+  - SS
```

Comment:
This is discouraged as it is too vague. Please use SFSS, DFSS (whichever is more appropriate)